package.json

```diff
@@ -11,6 +11,10 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
```

Comment:
Most of these dependencies should be in `devDependencies`, instead of `dependencies`.

Reply:
For a front-end JS project, yes. In our case, we're using NPM exclusively in the dev environment so it shouldn't really matter. But, I agree that it might make sense though to split these deps up so we can clearly see which ones we're including in our user scripts.

Reply:
Even in webdev environments, it doesn't matter. The `npm` client only pulls down `devDependencies` for the top level module (the one you type `npm install` in). 

It's about making it more readable for humans.